docs/site/pages/plugins/beacon.mdx

```diff
@@ -32,6 +32,8 @@ interface DefaultBeacon {
 }
 ```
 
+## Usage
```

Comment:
👍 this should be the standard nomenclature for all plugin docs, I think Expression is missing this

Reply:
https://github.com/player-ui/player/blob/0ad9807460e6cc5dd533264ef40e4d09cc959ed9/docs/site/pages/plugins/expression.mdx#L10

should be there for expression